solutionbox/structured_data/datalab_solutions/structured_data/preprocess/cloud_preprocess.py

```diff
@@ -0,0 +1,305 @@
+# Copyright 2017 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import argparse
+import os
+import sys
+import json
+
+import google.cloud.ml as ml
+
+
+INPUT_FEATURES_FILE = 'input_features.json'
+SCHEMA_FILE = 'schema.json'
+
+NUMERICAL_ANALYSIS = 'numerical_analysis.json'
+CATEGORICAL_ANALYSIS = 'vocab_%s.csv'
```

Comment:
These definitions are repeated in util,py, local_preprocess.py. Can we centralize them to util.py?